vault/dendron.topic.pod.builtin.markdown.import.md

```diff
@@ -92,8 +92,59 @@ If you set `indexName` to `index` (matching is case insensitive and doesn't requ
 
 In this example, the contents of `index.md` were combined with `foo.md`
 
+### importFrontmatter
+- default: false
```

Comment:
just a note that if we decide to change the default to true in the code to also update docs here.